recipe/meta.yaml

```diff
@@ -1,19 +1,19 @@
 # Full credit goes to https://github.com/conda/conda-recipes for providing this recipe.
 # It has been subsequently adapted for automated building with conda-forge.
 
-{% set version = "3.9.4" %}
+{% set version = "3.10.0" %}
 
 package:
   name: matplotlib-suite
   version: {{ version }}
 
 source:
   url: https://github.com/matplotlib/matplotlib/archive/v{{ version }}.tar.gz
-  sha256: a29f8bab72f81df9077480e6b55394a17effd329d0ff1524bb5bc570734e7c54
+  sha256: 6a921cd5c92ead9aab9c9c845865e27e3342eee8681eb36669c09f717efbff31
```

Comment:
The checksum changed. GH does that when the release is update upstream (that is the one advantage of PyPI, the checksum won't change).

 This is probably OK and the checksum just needs to the update, but I'll leave to the upstream devs here to confirm this b/c lately high profile packages are under attack of crypto miners.

Reply:
I merged the rc banch in...it is also possible that I screwed up the merge.

Reply:
You did nothing. GH needs to fix that situation. It is, IMO, bad that a released tag changes the checksum like that. Sure one could upload a tarball as an asset to the release and that checksum would not change, but then why to we have releases on GH? (Sorry for the rant.)